engines/ags/ags.cpp

```diff
@@ -70,6 +71,17 @@
 
 namespace AGS {
 
+struct AGSInstallerTableEntry {
+	const char *installerName;
+	const char *mainFileName;
+};
+
+static const AGSInstallerTableEntry agsInstallerTable[] = {
+	{"manicmdsetup.exe", "Maniac.exe"},
```

Comment:
I would personally stick it into `extra` in the detection and a game flag (GF_). Something like

```
GAME_ENTRY_INSTALLER(maniacmansiondeluxe", "manicmdsetup.exe",  "Maciac.exe", "329ff478ffc1a3bd0dc941d5212aa602", 5826271),

// And that specifies GF_INSTALLER
```

But that is a matter of taste. My reasoning is to keep it in a single place, so things do not get desync accidentally.

Reply:
Done